hazelcast-mesos-scheduler/src/main/java/com/hazelcast/mesos/util/HazelcastProperties.java

```diff
@@ -1,11 +1,15 @@
 package com.hazelcast.mesos.util;
 
+import java.net.Inet4Address;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
 import static com.hazelcast.mesos.util.Util.option;
 import static java.lang.Integer.parseInt;
 
 public class HazelcastProperties {
     static String HAZELCAST_VERSION = option("HAZELCAST_VERSION").or("3.6");
-    static String HOST = option("HOST").or("localhost");
+    static String HOST = option("HOST").or("MESOS_HOSTNAME");
```

Comment:
I didn't get why we are using two environment variables for the same purpose ? Is the content of `MESOS_HOSTNAME` variable provided by mesos or users ?